composer.json

```diff
@@ -17,6 +17,23 @@
     "ext-pdo": "*",
     "ext-json": "*"
   },
+  "require-dev": {
+    "phpunit/phpunit": "^4.8 || ^5.7 || ^6.5 || ^7.0"
+  },
+  "suggest": {
```

Comment:
Currently, we just support MySQL, PgSQL, Oracle and SQLite. Maybe we can check the other database drivers later, but now, you can remove other drivers except drivers that i mentioned.

Reply:
Ok. Leave other DB drivers, and keep `MySQL, PgSQL, Oracle and SQLite` existed on `suggest` block.